docs/_getting_started/jupyter.md

```diff
@@ -87,11 +216,14 @@ You can also launch interactive consoles from JupyterHub.  These will behave ver
 
```

Comment:
For all images, we should use alt text for accessibility reasons.  I recommend using chatgpt:

`![Terminal application icon with a dollar sign and underscore, categorized under 'Other'](path/to/image.png)`

Reply:
done